src/statement-list/converters/expr2statement_list.h

```diff
@@ -9,18 +9,137 @@ Author: Matthias Weiss, matthias.weiss@diffblue.com
 #ifndef CPROVER_STATEMENT_LIST_CONVERTERS_EXPR2STATEMENT_LIST_H
 #define CPROVER_STATEMENT_LIST_CONVERTERS_EXPR2STATEMENT_LIST_H
 
+#include <util/irep.h>
+#include <util/namespace.h>
+#include <util/std_expr.h>
+
 #include <string>
 
 /// Converts a given expression to human-readable STL code.
 /// \param expr: Expression to be converted.
 /// \param ns: Namespace of the TIA application.
 /// \result String with the STL representation of the given parameters.
-std::string expr2stl(const class exprt &expr, const class namespacet &ns);
+std::string expr2stl(const exprt &expr, const namespacet &ns);
 
 /// Converts a given type to human-readable STL code.
 /// \param type: Type to be converted.
 /// \param ns: Namespace of the TIA application.
 /// \result String with the STL representation of the given type.
-std::string type2stl(const class typet &type, const class namespacet &ns);
+std::string type2stl(const typet &type, const namespacet &ns);
+
+/// Class for saving the internal state of the conversion process.
+class expr2stlt
+{
+  /// Internal representation of the FC bit in STL. Used to track if the
+  /// current boolean operation is part of another.
+  bool inside_bit_string;
+
+  /// Flag to specify if the next symbol to convert is a reference to a
+  /// variable.
+  bool is_reference;
+
+  /// Contains the symbol table of the current program to convert.
+  const namespacet ns;
```

Comment:
Is there a specific reason you need to copy this? Otherwise prefer `const namespacet &ns`.